packages/react-app-scaffolder/app/templates/_package.json

```diff
@@ -12,20 +12,16 @@
   "private": true,
   "scripts": {
     <% if (isFoundation) { %>
-    <% if (stylesSolution === 'styledComponents') { %>
```

Comment:
move `build:prod`, `start:dev` out of if/else block when `isFoundation`, and use centralized build scripts